aptos-move/framework/aptos-token-objects/Move.toml

```diff
@@ -0,0 +1,13 @@
+[package]
```

Comment:
Why make this a new package vs merging into aptos-token? This would require making sure aptos-token-objects is supported as a new package in many places (genesis, release-tooling, etc.)

Reply:
Good call though... I need to actually run a local node and ensure that it is taken in otherwise this change won't impact devnet.

Reply:
I'd suggest ensuring both genesis and framework upgrade work. Otherwise, this would potentially bypass tests, compat checks, etc. which can somehow break the release process down the line

Reply:
A new package just because of shared names?

Reply:
I worry users would end up even more confused.